### PR TITLE
Cypress/fe 1737 maintenance for BDD specs

### PR DESCRIPTION
**Pull request by DlgSHi**
# maintenance for BDD specs
- fixes locators, steps

### CODE REVIEW

File: cypress/features/regression/icon.feature [hunk 1]
```diff
@@ -43,10 +43,11 @@ Feature: Icon component
       | ÄÖÜßäöüß                |
       | <>                      |
 
-  @positive
+  @ignore
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/icon.feature]: changed

File: cypress/features/regression/link.feature [hunk 2]
```diff
@@ -51,22 +51,37 @@ Feature: Link component
       | left      |
       | right     |
 
-  @positive
-  Scenario Outline: Change tooltip align to <tooltipAlign> and tooltipPosition to <tooltipPosition>
+  @ignore
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/link.feature]: changed

File: cypress/support/step-definitions/portrait-steps.js [hunk 3]
```diff
@@ -32,6 +20,12 @@ Then('Portrait source is set to {string}', (sourceProperty) => {
   }
 });
 
+Then('Portrait size is set to {string} and has {string}', (word, property) => {
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/portrait-steps.js]: fixed

File: cypress/features/regression/pill.feature [hunk 4]
```diff
@@ -21,17 +21,18 @@ Feature: Pill component
   @positive
   Scenario Outline: Set Pill as align to <as>
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/pill.feature]: fixed

File: cypress/features/regression/link.feature [hunk 5]
```diff
@@ -51,22 +51,37 @@ Feature: Link component
...
+      | top          |
+      | bottom       |
+      | center       |
+
+  @ignore
+  Scenario Outline: Change tooltip tooltipPosition to <tooltipPosition>
+    When I select icon to "add"
+      And I set tooltipMessage to "sample message"
+      And I select tooltipAlign to "center"
+      And I select tooltipPosition to "<tooltipPosition>"
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/link.feature]: fixed

File: cypress/features/regression/link.feature [hunk 6]
```diff
@@ -51,22 +51,37 @@ Feature: Link component
       | left      |
       | right     |
 
-  @positive
-  Scenario Outline: Change tooltip align to <tooltipAlign> and tooltipPosition to <tooltipPosition>
+  @ignore
+  Scenario Outline: Change tooltip align to <tooltipAlign>
     When I select icon to "add"
       And I set tooltipMessage to "sample message"
       And I select tooltipAlign to "<tooltipAlign>"
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/link.feature]: fixed

File: cypress/features/regression/link.feature [hunk 7]
```diff
@@ -51,22 +51,37 @@ Feature: Link component
       | left      |
       | right     |
 
-  @positive
-  Scenario Outline: Change tooltip align to <tooltipAlign> and tooltipPosition to <tooltipPosition>
+  @ignore
+  Scenario Outline: Change tooltip align to <tooltipAlign>
     When I select icon to "add"
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/link.feature]: fixed

File: cypress/features/regression/icon.feature [hunk 8]
```diff
@@ -56,10 +57,11 @@ Feature: Icon component
       | top             |
       | bottom          |
 
-  @positive
+  @ignore
   Scenario Outline: Change tooltip align to <tooltipAlign>
     When I set tooltipMessage to "sample message"
       And I select tooltipAlign to "<tooltipAlign>"
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/icon.feature]: fixed

File: cypress/features/regression/link.feature (hunk 6 above)
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/features/regression/link.feature]: ```And I select tooltipAlign to "<tooltipAlign>"``` this step is when

File: cypress/features/regression/link.feature (hunk 5 above)
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/features/regression/link.feature]: ```And I select tooltipPosition to "<tooltipPosition>"``` this step is when

File: cypress/support/step-definitions/pill-steps.js [hunk 9]
```diff
@@ -4,26 +4,27 @@ const PILL_AS_PROPERTY = 'carbon-pill--';
...
 const PILL_ON_DELETE_PROPERTY = 'carbon-pill--is-deletable';
+const BORDER_SOLID_VALUE = '1px solid ';
 
 Then('Pill children on preview is set to {string}', (text) => {
   pillPreview().should('have.text', text);
 });
 
-Then('Pill as on preview is {string}', (asProperty) => {
-  pillPreview().should('have.class', `${PILL_AS_PROPERTY}${asProperty}${PILL_EMPTY_PROPERTY}`);
+Then('Pill as on preview is {string} and has {string}', (asProperty, pillColor) => {
```
Comment (marcinstachowicz) [inline cypress/support/step-definitions/pill-steps.js]: asProperty is never used

File: cypress/support/step-definitions/portrait-steps.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (marcinstachowicz) [reply cypress/support/step-definitions/portrait-steps.js]: discussed :) word is never used

File: cypress/features/regression/link.feature (hunk 5 above)
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/link.feature]: fixed

File: cypress/features/regression/icon.feature [hunk 10]
```diff
@@ -56,10 +57,11 @@ Feature: Icon component
       | top             |
       | bottom          |
 
-  @positive
+  @position
```
Comment (marcinstachowicz) [inline cypress/features/regression/icon.feature]: position?

Comment (DlgSHi) [reply cypress/features/regression/icon.feature]: fixed

**Pull request merged**